openapi/src/parts/azure_landing_zone.yaml

```diff
@@ -189,6 +189,10 @@ components:
       type: object
       required: [definition, billingProfileId]
       properties:
+        landingZoneId:
+          description: The ID of the landing zone (optional)

```

Comment:
It would be good to describe more how this would be used. And if the `landingZoneId` is specified, then the `definition` is _not_ used, correct? Is `definition` still a required property in that case?

Reply:
Definition is required and always used as it specifies what type of landing zone to create (i.e, CromwellBaseResources in almost all cases); if the ID is omitted (as it is today) one is auto generated.

Reply:
I was assuming here that if the ID is specified, then attachment is done to an existing landing zone. That's not what is going on?

Reply:
No. We need to be able to specify the ID for the landing zone from test code as the pre deployed landing zone resources will already be tagged with that ID.